docs/source/settings.rst

```diff
@@ -1213,7 +1213,7 @@ The type of workers to use.
 The default class (``sync``) should handle most "normal" types of
 workloads. You'll want to read :doc:`design` for information on when
 you might want to choose one of the other worker classes. Required
-libraries may be installed using setuptools' ``extra_require`` feature.
+libraries may be installed using setuptools' ``extras_require`` feature.
```

Comment:
This file is automatically generated. Please make this change in `gunicorn/config.py` instead and regenerate this file by running `make -C docs/ html`.

Reply:
I need to regenerate this. I'll let you know when it's done. Thanks